custom_code/alertstream_handlers.py

```diff
@@ -301,9 +300,26 @@ def handle_einstein_probe_alert(message, metadata):
             skymap['PROBDENSITY'].unit = '1 / sr'
             calculate_credible_region(skymap, localization)
             calculate_footprint_probabilities(skymap, localization)
+    
+    ep_ra = alert.get('ra')
+    ep_dec = alert.get('dec')
+    ep_name = alert['id'][0]
+    ep_error = alert.get('ra_dec_error')
+    t_ep = Target.objects.create(name=ep_name, type='SIDEREAL', ra = ep_ra, dec = ep_dec)
+
+    # send SMS, Slack, and email alerts
+    ALERT_TEXT_URL_EP = "https://sand.as.arizona.edu/saguaro_tom/targets/{t_ep.id}/"
```

Comment:
Here it would be slightly better to use `settings.TARGET_LINKS[0][0]` in case we want to send this to DLT40 in the future.